worker/restore.go

```diff
@@ -194,7 +194,7 @@ func loadFromBackup(db *badger.DB, in *loadBackupInput) (uint64, uint64, error)
 					return 0, 0, errors.Wrapf(err, "while reading backup posting list")
 				}
 				pl := posting.FromBackupPostingList(backupPl)
-				shouldSplit := pl.Size() >= (1<<20)/2 && len(pl.Pack.Blocks) > 1
+				shouldSplit := pl.Size() >= (1<<20)/2
```

Comment:
should we check the len(bitmap) over here? Similar to how we were checking len(pl.Pack.Blocks)?

Reply:
It is not strictly required. We call rollups if `shouldSplit` is `true`, which will then be a noop as it checks for the cardinality. 
Plus, checking for cardinality of a bitmap is expensive (we will have to de-serialize the bitmap). So I think that the check on posting list size can act as a preliminary filter here.

Reply:
Done

Reply:
Also add this check in bulk loader.